src/source/from-shift.js

```diff
@@ -0,0 +1,205 @@
+'use strict';
+
+function makeEager(obj) {
+	obj.type = 'Eager' + obj.type;
+}
+
+function dummyDeclaredScope(type) {
```

Comment:
I have the impression that the indentations are much larger than anywhere in the project. Is it me?

I don't really care all that much, but consistency is good :)

Reply:
I used tabs here as in other JS files, but yeah, it was me who created other JS files too with different indentation from Rust :/

I don't mind reformatting to spaces if you prefer, but diff will get somewhat larger.